### DISCUSSION TITLE
How to Group Objects that are loaded by loadFromJson() function ?

### DISCUSSION THREAD

**Discussion by vipin-vidyut** [fabricjs/fabric.js#8575] [General]
How to Group Objects that are loaded by loadFromJson() function ?
`var json = '{"objects":'+JSON.stringify(attr.objects)+'}';
            // console.log(json);
            canvas.loadFromJSON(json, canvas.renderAll.bind(canvas), function(o, object) {
              // `o` = json object
              // `object` = fabric.Object instance
              // ... do some stuff ...     
            });`

**Comment by ShaMan123** [comment]
use `enlivenObjects` directly instead.
